src/blameandshame/util.py

```diff
@@ -218,3 +218,19 @@ def lines_modified_by_commit(repo: git.Repo,
                 new_line_num += 1
 
     return (frozenset(old_lines), frozenset(new_lines))
+
+
+def last_commit_to_line(repo: git.Repo,
+                        filename: str,
+                        lineno: int,
+                        before: git.Commit) -> Optional[git.Commit]:
+    """
+    Returns a Commit object corresponding to the last commit where lineno was
+    touched before (and including) the Commit object passed in before.
+    """
+    try:
+        commits = commits_to_line(repo, filename, lineno, None, before)
+    except git.exc.GitCommandError:
+        commits = [None]
```

Comment:
Should be `commits = None`.

Reply:
This is so that I can use `return commits[0]` instead of `return None if commits == None else commits`

Reply:
Oh, I see. That's fine.